editions/test/tiddlers/tests/test-filters.js

```diff
@@ -14,6 +14,22 @@ Tests the filtering mechanism.
 /* global $tw, require */
 "use strict";
 
+// This wrapper method is used to collect warnings which should be emitted
+// by certain deprecated tests.
+function collectLog(block) {
+	var messages = [];
+	var oldLog = console.log;
+	console.log = function(a) {
+		messages.push(Array.prototype.join.call(arguments, ' '));
```

Comment:
A very small point, but we use double quotes for string literals (except for a few cases where the string contains double quotes).

Reply:
Switched over. Should be good to merge?